addons/web/static/src/views/list/list_renderer.xml

```diff
@@ -288,7 +288,7 @@
             <t t-if="hasOpenFormViewColumn">
                 <td class="o_list_record_open_form_view w-print-0 p-print-0 text-center"
                     t-on-keydown="(ev) => this.onCellKeydown(ev, group, record)"
-                    t-custom-click.stop="(ev, newWindow) => isX2Many and record.isNew ? this.displaySaveNotification() : props.onOpenFormView(record, { force: true, newWindow })"
+                    t-custom-click.stop="(ev, newWindow) => this.openFormView(record, newWindow)"
```

Comment:
there's another usecase for displaySaveNotification, which is when the user clicks on a regular button in the list view. This one should be fixed as well, and we can get rid of the `dispaySaveNotification` function.